applications/nrf_desktop/doc/smp.rst

```diff
@@ -33,12 +33,12 @@ Make sure that the following Kconfig options are enabled:
   See the :ref:`zephyr:device_mgmt` for details about Zephyr's device management subsystem.
 * :option:`CONFIG_MCUMGR_SMP_BT`
   Device receives SMP commands over Bluetooth.
-* ``CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE``
```

Comment:
The `CONFIG_DESKTOP_...` Kconfig options are specific to nrf_desktop project.
Their documentation pages are not generated in the documentation. Therefore its not possible to refer to these options using `:option:`. We have to leave them the with the previous syntax.

Reply:
I will revert this part of your change

Reply:
Thanks for the note, fixed